lib/brakeman/util.rb

```diff
@@ -124,6 +124,18 @@ def hash_access hash, key
     nil
   end
 
+  def call_has_param call, key
```

Comment:
Yes, sorry. You can check with `#method`.

Something like:

```ruby
if call? arg and call? arg.target and node_type? arg.target.target, :params
  arg.target.method == :to_unsafe_hash or arg.target.method == :to_unsafe_h
end
```

This is just an example. It would probably be nicer to use some variables in there to clarify what the different pieces are.